src/slices/comments.slice.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { getPostComments } from '../api/comments';
+
+type Comments = {
+  loaded: boolean;
+  hasError: boolean;
+  items: Comment[];
+};
+
+const initialState: Comments = {
+  loaded: true,
```

Comment:
The initial state for 'loaded' should be false, as the comments are not loaded initially. Setting it to true might cause confusion as it indicates that the comments are already loaded when the application starts.